src/inspector_agent.cc

```diff
@@ -288,6 +288,7 @@ class V8NodeInspector : public blink::V8Inspector {
       return;
     terminated_ = false;
     running_nested_loop_ = true;
+    agent_->DispatchMessages();
```

Comment:
Is this thread-safe?  DispatchMessages() reads and writes dispatching_messages_ without holding any locks.